Core/include/Acts/EventData/SpacePointContainer.hpp

```diff
@@ -22,41 +22,21 @@ namespace Acts {
 
 struct SpacePointContainerConfig {
   bool useDetailedDoubleMeasurementInfo = false;
-  bool isInInternalUnits = false;
-
-  SpacePointContainerConfig toInternalUnits() const {
-    if (isInInternalUnits) {
-      throw std::runtime_error(
-          "Repeated conversion to internal units for "
-          "SpacePointContainerConfig");
-    }
-    using namespace Acts::UnitLiterals;
-    SpacePointContainerConfig config = *this;
-    config.isInInternalUnits = true;
-    return config;
-  };
+
+  bool isInInternalUnits = true;
```

Comment:
Why is this still needed?

Reply:
I left this in to not break the API for now. We can do this with the next major release